docker/1.4.4/config/mosquitto.conf.default

```diff
@@ -0,0 +1,819 @@
+# Config file for mosquitto
+#
```

Comment:
This is a clone of the mosquitto.conf.example file right?  Why does it need to be a copy here?  It makes it far more work to update any config options if you have to keep this file up todate too.